op-node/rollup/engine/engine_controller.go

```diff
@@ -45,26 +46,23 @@ type ExecEngine interface {
 	L2BlockRefByHash(ctx context.Context, hash common.Hash) (eth.L2BlockRef, error)
 }
 
-type ECMetrics interface {
-	derive.Metrics
-	RecordL2Ref(name string, ref eth.L2BlockRef)
-}
-
 type SyncDeriver interface {
 	OnELSyncStarted()
 }
 
 type EngineController struct {
 	engine     ExecEngine // Underlying execution engine RPC
 	log        log.Logger
-	metrics    ECMetrics
+	metrics    opmetrics.Metricer
 	syncCfg    *sync.Config
 	syncStatus syncStatusEnum
 	chainSpec  *rollup.ChainSpec
 	rollupCfg  *rollup.Config
 	elStart    time.Time
 	clock      clock.Clock
 
+	// Event system fields (moved from EngDeriver)
```

Comment:
Can we add a comment that this comment is related to event system refactor and eventually be removed?
```go
// TODO(#16917) Remove Event System Refactor Comments
```